addons/mail/static/src/components/rtc_call_participant_card/rtc_call_participant_card.scss

```diff
@@ -109,30 +115,30 @@
 
 .o_RtcCallParticipantCard {
     border-radius: $o-mail-rounded-rectangle-border-radius-sm;
+    cursor: pointer;
```

Comment:
Focus is disabled when minimized, so the pointer is misleading in that case as the click does nothing.

Reply:
Except if it's an invitation card like before, that can always be clicked with an effect.